catalogs/catalog-jdbc-doris/src/main/java/org/apache/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -221,7 +221,7 @@ private static void validateDistribution(Distribution distribution, JdbcColumn[]
 
     Preconditions.checkArgument(
         Strategy.HASH == distribution.strategy() || Strategy.EVEN == distribution.strategy(),
-        "Doris only supports HASH or EVEN distribution strategy");
+        "Doris only supports HASH or EVEN(RANDOM) distribution strategy");
```

Comment:
if the `Strategy` is `EVEN`, we should check the `distribution.expressions()` must be empty?